azure/ConsiderationsForServiceDesign.md

```diff
@@ -332,74 +406,77 @@ HTTP/1.1 200 OK
 }
 ```
 
-### PUT with additional long-running processing
+This diagram illustrates how a long-running action operation is initiated and then how the client
+determines it has completed and obtains its results:
 
-A special case of long-running operation that occurs often is a PUT operation to create or replace a resource
-that involves some additional long-running processing.
-One example is a resource requires physical resources (e.g. servers) to be "provisioned" to make the resource functional.
-In this case, the request may contain an `Operation-Id` header that the service will use as
-the ID of the status monitor created for the operation.
+```mermaid
+sequenceDiagram
+    participant Client
+    participant API Endpoint
+    participant Status Monitor
+    Client->>API Endpoint: POST
+    API Endpoint->>Client: HTTP/1.1 202 Accepted<br/>{ "id": "22", "status": "NotStarted" }
+    Client->>Status Monitor: GET
+    Status Monitor->>Client: HTTP/1.1 200 OK<br/>Retry-After: 5<br/>{ "id": "22", "status": "Running" }
+    Client->>Status Monitor: GET
+    Status Monitor->>Client: HTTP/1.1 200 OK<br/>{ "id": "22", "status": "Succeeded", "result": { ... } }
+```
 
-```text
-PUT /items/FooBar&api-version=2022-05-01
-Operation-Id: 22
+1. The client sends the request to initiate the long-running action operation.
+The request may contain an `Operation-Id` header that the service uses as the ID of the status monitor created for the operation.
 
-{
-   "prop1": 555,
-   "prop2": "something"
-}
-```
+2. The service validates the request and initiates the operation processing.
+If there are any problems with the request, the service responds with a `4xx` status code and error response body.
+Otherwise the service responds with a `202-Accepted` HTTP status code.
+The response body is the status monitor for the operation including the ID, either from the request header or generated by the service.
+When returning a status monitor whose status is not in a terminal state, the response must also include a `retry-after` header indicating the minimum number of seconds the client should wait
+before polling (GETing) the status monitor URL again for an update.
+For backward compatibility, the response may also include an `Operation-Location` header containing the absolute URL
+of the status monitor resource (without an api-version query parameter).
```

Comment:
This won't render correctly. If you break on a numbered list, you must shift subsequent text to where the first line's text begins (2 spaces for `* ` or `- `, 3 spaces for `1. `, etc.

```suggestion
2. The service validates the request and initiates the operation processing.
   If there are any problems with the request, the service responds with a `4xx` status code and error response body.
   Otherwise the service responds with a `202-Accepted` HTTP status code.
   The response body is the status monitor for the operation including the ID, either from the request header or generated by the service.
   When returning a status monitor whose status is not in a terminal state, the response must also include a `retry-after` 
   header indicating the minimum number of seconds the client should wait
   before polling (GETing) the status monitor URL again for an update.
   For backward compatibility, the response may also include an `Operation-Location` header containing the absolute URL
   of the status monitor resource (without an api-version query parameter).
```

You'll need to fix this for all numbered items you modified.

Reply:
Or, just quit trying to make so many new lines and let the text flow. It's all going to wrap in raw and rendered HTML text anyway. So much easier to make changes in the long run.

Reply:
Actually I didn't make these changes and the original seems to format correctly. So I will leave it alone for now and we can check it again in the next round of review.

Reply:
GitHub must use a relaxed markdown renderer then. It's actually a violation of CommonMark.